src/serde/ser_br.rs

```diff
@@ -77,10 +78,45 @@ pub fn node_to_stream_backrefs<W: io::Write>(
     Ok(())
 }
 
+pub fn node_to_bytes_backrefs_limit(
+    a: &Allocator,
+    node: NodePtr,
+    limit: usize,
+) -> io::Result<Vec<u8>> {
+    let buffer = Cursor::new(Vec::new());
+    let mut writer = LimitedWriter::new(buffer, limit);
+    node_to_stream_backrefs(a, node, &mut writer)?;
+    let vec = writer.into_inner().into_inner();
+    Ok(vec)
+}
+
 pub fn node_to_bytes_backrefs(a: &Allocator, node: NodePtr) -> io::Result<Vec<u8>> {
-    let mut buffer = Cursor::new(Vec::new());
+    node_to_bytes_backrefs_limit(a, node, 2000000)
+}
```

Comment:
I don't think you should change this function. it should not have a default limit.

Reply:
Right. I reverted my code.